src/components/project_admin/instructor_files/single_instructor_file.vue

```diff
@@ -187,38 +158,50 @@ export function handle_rename_file_error(component: SingleInstructorFile, error:
 .not-editing {
   display: flex;
 
-  .file-name {
-    font-size: 16px;
-    padding-bottom: .125rem;
+  .select-checkbox {
+    width: 1.35rem;
+    transform: scale(1.35);
   }
 
-  .edit-file-name, .download-file, .delete-file {
-    margin: 0 .375rem;
-  }
+  .file-info-actions {
+    display: flex;
+    justify-content: space-between;
+    width: 90%;
```

Comment:
If I remove the `file-info-actions` div and css class (and the `display`, `justify-content`, and `width` rules here), I don't see a difference in the rendered format. Am I missing something, or can this be removed?

Reply:
Looks like this for me with those attributes removed:

<img width="268" alt="Screen Shot 2020-12-18 at 6 54 48 PM" src="https://user-images.githubusercontent.com/16354853/102674043-92c64580-4162-11eb-9558-788bba9446d6.png">

The width 90% is to allow room for scrollbar. With this width, the icon is out of the way of the scroll bar but not enough to look visibly odd. Totally up to you whether you want to keep that, I just thought it worked better on a longer list of files.

Reply:
Got it. It looks fine either way in my browser (non-mac), so we'll keep it.